tutorials/ai-core-azure/ai-core-azure.md

```diff
@@ -0,0 +1,79 @@
+---
+author_name: Dhrubajyoti Paul
+author_profile: https://github.com/dhrubpaul
+title: Setup Azure Bucket With AI Core
+description: In this tutorial we will be walking through the steps on how to Set up azure bucket (Azure Blob Storage) with SAP AI Core
+keywords: Azure Bucket setup
+auto_validation: true
+time: 10
+tags: [ tutorial>beginner, topic>artificial-intelligence, topic>machine-learning, software-product>sap-business-technology-platform, software-product>sap-ai-launchpad, software-product>sap-ai-core ]
+primary_tag: software-product>sap-ai-core
+parser: v2
+---
+# Setup Azure DataBucket With AI Core
+<!-- description -->Set up azure bucket (Azure Blob Storage) with SAP AI Core.
+
+## Prerequisites
+- An existing Azure account with Admin access to create and use Azure Datastore.
+-TODO establish the setup prerequisites for SAP AI Core 
+![image](img/azure-dashboard.png)
+
+## You will learn
+- About Azure data store
+- To setup Azure for use with AI core
+- Setup Azure data buckets for use with AI Core.
+
+## What is azure data store ?
+
+>**IMPORTANT** Before you start this tutorial with SAP AI Launchpad, it is recommended that you should set up at least one other tool, either Postman or Python (SAP AI Core SDK) because some steps of this tutorial cannot be performed with SAP AI Launchpad.
+
+
+### System Setup
+For Detailed docs reffer : [AI Core Setup](https://developers.sap.com/tutorials/ai-core-setup.html)
+TODO establish the setup prerequisites for SAP AI Core 
+
+
+- Downloading postman config
```

Comment:
I recommend using a numbered sequence for this and the following steps.
"Downloading" should be an active "Download."  Remember that you are directing the user through steps.